Ms2Database/Controllers/CharacterManager.cs

```diff
@@ -42,11 +57,11 @@ public Character GetCharInfo(long charId)
             }
         }
 
-        public void SetCharInfo(Character CharIn)
+        public void EditCharInfo(Character character)
         {
             using (Ms2DbContext context = new Ms2DbContext())
             {
-                Character character = CharIn;
+                Character OriginCharacter = character;
```

Comment:
why this naming convention?

Reply:
Figured I'd name the this.variable to something more fitting since param is the same name

Reply:
Prob change it to something different